docs/en/usage.md

```diff
@@ -0,0 +1,103 @@
+## Workflow
+
+1. Upload the original video file to the [Amazon S3][s3] bucket.
+2. Initiate a processing request through [Amazon API Gateway][api-gateway] (see below for request parameters).
+3. The [Amazon Lambda][lambda] function receives user requests, reads the metadata of the original video file from the S3 bucket, and calculates the number of blocks, then submits the calculation job of [Amazon Batch][Batch].
+4. [Amazon Batch][Batch] pulls the pre-built model image from ECR, and launches the [Amazon EC2 Inf1][inf1] instance through ECS to run the following calculation Job:
+     1. Video slicing: download the original video from the S3 bucket and split it into several slices. Currently, every minute of the video is considered as one slice.
+     2. Super-resolution processing: run a Job for each slice, and perform super resolution on each slice of the video based on the pre-trained model.
+     3. Merging: the super resolution results are merged, and then the merged result file is uploaded to S3.
+
+## Request parameters
+
+| Parameter | Default | Description |
+|---|---|---|
+| key | \<Requires input\> | Indicates the file name in S3.  |
+| scale | 2 | Defines super resolution scaling (one-sided). The allowed values are 2 or 4. |
+| task | inference | Specifies the task type. The allowed values are debug or inference. When it is set to debug, all intermediate files will be uploaded to S3 for debugging purposes.  |
+| env | spot | Specifies the Job running environment. The allowed values are onDemand or spot. |
+
+## Example of request code
+
+The following example shows the request code after the deployment with default configurations.
```

Comment:
such format "https://xxxxx.execute-api.xxxxx.amazonaws.com/prod" is for non ZHY/BJS region, explicit noted